### PR TITLE
Internal: JS API - Changes to 'settings' model should be via internal command [ED-3715]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: assets/dev/js/editor/document/globals/commands/disable.js [hunk 1]
```diff
@@ -34,7 +34,15 @@ export class Disable extends DisableEnable {
...
 
-			container.settings.set( '__globals__', container.globals.toJSON() );
+			$e.internal( 'document/elements/set-settings', {
+				container,
+				settings: {
+					__globals__: container.globals.toJSON(),
+				},
+				options: {
+					render: false,
+				},
```
Comment (matipojo) [inline assets/dev/js/editor/document/globals/commands/disable.js:44]: How it worked until now?

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/document/globals/commands/disable.js (hunk 1 above)
Comment (iNewLegend) [reply assets/dev/js/editor/document/globals/commands/disable.js:44]: Next line `container.renderUI()` you want to add it into `set-settings` as options?

Comment (matipojo) [reply assets/dev/js/editor/document/globals/commands/disable.js:44]: yes.

Comment (iNewLegend) [reply assets/dev/js/editor/document/globals/commands/disable.js:44]: Handled.

File: assets/dev/js/editor/document/elements/commands-internal/set-settings.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import CommandInternal from 'elementor-api/modules/command-internal-base';
+
+export class SetSettings extends CommandInternal {
+	validateArgs( args = {} ) {
+		this.requireContainer( args );
+		this.requireArgumentType( 'settings', 'object', args );
+
+		if ( 'undefined' !== typeof args.render && 'undefined' !== typeof args.renderUI ) {
+			throw new Error( "Args: 'render' and 'renderUI' cannot be applied together." );
```
Comment (matipojo) [inline assets/dev/js/editor/document/elements/commands-internal/set-settings.js:9]:
```suggestion
			throw new Error( 'Args: `render` and `renderUI` cannot be applied together.' );
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**